website/content/configuration/_index.md

```diff
@@ -88,19 +88,47 @@ data:
       - 192.168.10.0/24
 ```
 
+By default, BGP speaker will announce IPv4 prefixes as specified in rfc4271. 
+For IPv6 prefixes Multiprotocol extension for BGP-4 (rfc4271) will be used.
+ If peer supports Multiprotocol encoding for IPv4 you can allow MetalLB to use it via `allow-mp-bgp-encoding-ipv4` option.
```

Comment:
Is this default `true` or default `false`. Default true seems a better idea since any remotely modern router will handle BGP-MP.

Reply:
By default, it is `false`.  [Here ](https://github.com/metallb/metallb/pull/590#issuecomment-617023344) I try to describe why. If we change default encoding to MP-BGP we can break compatibility with some peers. For example, Bird daemon 1.6.x (this version is shipped with current LTS Ubuntu right now) will stop working. I think it can break compatibility not only with bird daemon, but with many other BGP implementations.  I think default settings should provide as much backward compatibility as we can provide.

Reply:
I did not know bird misbehaved in this way. Makes total sense now.